Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7377,6 +9515,20 @@ def main():
         elif demisto.command() == 'panorama-install-file-content-update':
             panorama_install_file_content_update_command(args)
 
+        # The following code exists PURELY to supress XSOAR linter errors. This code is
+        # unreachable as these commands are routed via the COMMAND object.
+        elif demisto.command() == 'pan-os-platform-get-arp-tables':
+            run_normal_command(get_arp_tables, demisto.args())
```

Comment:
I've changed this to make it more simplistic. We still have a function for converting the dataclasses into CommandResults, but otherwise it's a more procedural flow of code.

Reply:
awesome work Adam!!!